util/config.go

```diff
@@ -119,6 +119,11 @@ func (cfg *Config) WithTokens(accessTokenType, accessToken, refreshToken, idToke
 	return cfg
 }
 
+/* Return the ID token, or "" if does not exist. */
+func (cfg *Config) GetIdToken() string {
```

Comment:
idiomatic go doesn't like getters. It also doesn't have a problem with accessing public structure members directly, like we do for the access token in cli,InitCtx(). SInce we only get the idToken in one place, I'd do it like that rather than proliferate accessors. Suggest IdToken() if you really want to keep this func.